src/mlpack/tests/CMakeLists.txt

```diff
@@ -68,7 +69,6 @@ add_executable(mlpack_test
   matrix_completion_test.cpp
   maximal_inputs_test.cpp
   mean_shift_test.cpp
-  metric_test.cpp
```

Comment:
Can you put `metric_test.cpp` again for keeping the alphabetical order?

Reply:
Sorry about that, changing it.